tests/integration/kernel/src/main.c

```diff
@@ -302,6 +306,38 @@ ZTEST(kernel, test_mutex_usage)
 	k_thread_abort(tid2);
 }
 
+static void high_prio_main(void *param1, void *param2, void *param3)
```

Comment:
msgq tests should go into `tests/kernel/msgq/`

Reply:
Hi @nashif ,
This test include two behavior `poll`, `message` and require run on only 1 cpu so I think it is better here than `msgq`.
WDYT?

Reply:
can also go into `tests/kernel/poll/`, `tests/integration/kernel` is not the right place, we need this per module/kernel feature and all in one place related to the primitive being changed.

Reply:
Done.